### DISCUSSION TITLE
Add the ability to ban VPNs/specific ip providers/datacentres from connecting to servers

### DISCUSSION THREAD

**Discussion by trancer1994** [BearWare/TeamTalk5#2076] [Q&A]
### Description

Hi,

We run a server which is in the unofficial server list, and recently we have been having an influx of spammers using vpns to connect to the server, which they then spam with, and even when they get banned they still connect using different ips, possibly on the same vpn. Would you be able  to introduce the ability for moderators of servers to be able to ban specific VPNs from connecting to those servers? These vpns use specific ip ranges which don't change often, so this would be very easy to implement. 

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [X] TeamTalkServer

**Comment by bear101** [comment]
https://bearware.dk/?p=1433
